docs/api/config.md

```diff
@@ -1,99 +1,152 @@
+
+<div class="alert alert--info" role="alert" style="margin-top: 10px;">
+  The target directory <em>must</em> contain an index.html file.
+</div>`},
+{property: "devPath", type: "string", description: `Can be a path—either absolute or relative—to a folder or a URL (like a live reload server).`},
+{property: "beforeDevCommand", type: "string", description: `A command to run before starting Tauri in dev mode.`},
+{property: "beforeBuildCommand", type: "string", description: `A command to run before starting Tauri in build mode.`}
+]}/>
 
-```js
+```js title=Example
 "build": {
   "distDir": "../dist",
   "devPath": "http://localhost:4000",
-  "beforeDevCommand": "",
-  "beforeBuildCommand": ""
+  "beforeDevCommand": "npm run dev",
+  "beforeBuildCommand": "npm run build"
 }
 ```
 
-### `distDir`
-
-Type: `string`
-
-The path—either absolute or relative—to the production-ready webpage/webapp directory that will be bundled by Tauri.
-
-<div className="alert alert--info" role="alert">
-  The target directory <em>must</em> contain an index.html file.
-</div>
-
-### `devPath`
-
-Type: `string`
-
-Can be a path - either absolute or relative - to a folder or a URL (like a live reload server).
-
-### `beforeDevCommand`
-
-Type: `string`<br/>
-Default: `""`
-
-A command to run before starting Tauri in dev mode.
-
-### `beforeBuildCommand`
-
-Type: `string`<br/>
-Default: `""`
+## `ctx`
 
-A command to run before starting Tauri in build mode.
+<Properties rows={[
+  { property: "debug", type: "boolean", description: `Tells the CLI to build on debug mode instead of release. Can also be used as <code>tauri build --debug</code>.` },
+  { property: "exitOnPanic", type: "boolean", description: `Whether or not the webview should be killed if some Rust code error happened. Can also be used as <code>tauri dev --exit-on-panic</code> or <code>tauri dev -e</code>.` }
+]} />
 
-## ctx
-
-```js
+```js title=Example
 "ctx" {
-  "prod": false,
-  "dev": false,
-  "target": "",
   "debug": false,
-  "targetName": "",
   "exitOnPanic": false,
 }
 ```
 
-### `prod`
-
-Type: `boolean`<br/>
-Default:
-
-### `dev`
-
-Type: `boolean`<br/>
-Default:
-
-### `target`
-
-Type: `string`<br/>
-Default:
-
-### `debug`
-
-Type: `boolean`<br/>
-Default:
-
-### `targetName`
+## `tauri`
 
-Type: `string`<br/>
-Default:
-
-### `exitOnPanic`
-
-Type: `boolean`<br/>
-Default:
+<Properties rows={[
+  {
+    property: "embeddedServer", type: "object",
+    child: <Properties rows={[{ property: "active", type: "boolean", description: `Set it to <code>false</code> if you plan to serve your application statically.` }]} />
+  },
+  {
+    property: "bundle", type: "object",
+    child: <Properties rows={[
+      { property: "active", type: "boolean", description: `` },
+      { property: "targets", type: "string", description: `The name of the built application. If this is not present, then it will use the name value from your Cargo.toml file.` },
```

Comment:
`targets` is an array of the bundles you want to generate; e.g. ["deb", "osx", "msi", "appimage", "dmg"]. By default we bundle everything your target supports (osx/dmg on mac, deb/appimage on linux, msi on windows).